### DISCUSSION TITLE
Desktop not Seeing VM on Mac

### DISCUSSION THREAD

**Discussion by JJHWAN** [podman-desktop/podman-desktop#406] [Q&A] [answered]
I have made dockerd VM with Lima docker [example](https://github.com/lima-vm/lima/blob/master/examples/docker.yaml) 

After I switched context to lima-docker with followings in the docker.yaml, I expected Podman Desktop to show me the Docker extension since I have experienced with Rancher Desktop.

But Nothing seems to changed. Is there specific condition needed to be seen in Podman Desktop Docker extension?

**Comment by benoitf** [comment, marked as answer]
Hello @JJHWAN 

I think we support only the lima/podman and not lima/docker
I think socket for lima/Docker is at a different location while we search only for the 'podman' one

https://github.com/containers/podman-desktop/blob/main/extensions/lima/src/extension.ts#L45-L46

But it does not seems complicated to add support for lima/docker in addition to lima/podman

**Reply by JJHWAN** [reply]
Thanks for quick reply! @benoitf 
It seems to be possible with the addition for the lima/Docker
But I'm still curious about why rancher desktop context was caught on Podman Desktop and why lima/Docker can't

Can you tell me how Podman Desktop get's docker daemon's socket path?
Does it just ping with /var/run/docker?

**Reply by benoitf** [reply]
Docker usually adds socket into /var/run/docker.sock
so yes Docker extension in Podman Desktop tracks this usage so when an engine adds/uses this path it's detected automatically

For lima it's using a different path so it's not yet handle
